shared/test/resources/tests.yaml

```
@@ -1501,3 +1501,9 @@
 "Þi":
   - {in: [[1,[2,3]],[1,0]], out: 2}
   - {in: [["xyzabc"], [0,4]], out: "b"}
+
+"øh":
```

```suggestion
"Þh":
```
One last thing and then I'll approve